client/rest-high-level/src/test/java/org/elasticsearch/client/indices/AnalyzeResponseTests.java

```diff
@@ -19,25 +19,24 @@
 
 package org.elasticsearch.client.indices;
 
+import com.carrotsearch.randomizedtesting.annotations.Repeat;
+
 import org.elasticsearch.action.admin.indices.analyze.AnalyzeAction;
 import org.elasticsearch.client.AbstractResponseTestCase;
 import org.elasticsearch.common.xcontent.XContentParser;
 
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
 
+@Repeat(iterations = 100)
```

Comment:
Do you want to keep this or was it just for debugging?

Reply:
Also gone.